errorDisplay.js

```diff
@@ -8,60 +8,99 @@ define(function (require, exports, module) {
         ExtensionUtils = brackets.getModule("utils/ExtensionUtils"),
         lineWidgetHTML = require("text!inlineWidget.html"),
         currentErrorWidget;
-
+    var errorToggle;
```

Comment:
It's really important to be consistent in how you change code.  If a file has an existing style, you should match it.  In this case, you are mixing single `var` and `var`-per-line, which is odd.